src/screens/Inventory/CreateInventoryUpdate.tsx

```diff
@@ -19,6 +21,10 @@ const styles = (theme: Theme) =>
     },
   });
 
+const validationSchema = yup.object({
+  updatedAmount: yup.number().positive('Please enter a positive number').required('Please enter an amount'),
```

Comment:
NIT: Maybe it'd be good to allow 0?

![image](https://user-images.githubusercontent.com/35501399/115128230-2bde5f00-9f91-11eb-83ed-6dd21662f08e.png)